app/utils/file.js

```diff
@@ -75,3 +75,6 @@ export function buildFileUploadData(file) {
         extension
     };
 }
+
+export const encodeHeaderURIStringToUTF8 = (string) =>
```

Comment:
Can you add brace brackets around this? I think we should require them when the function is on two lines like this

Reply:
I'm approving the PR cause your going to make the change requested by HH ;)

Reply:
Whatevs. ;) I like them Implicit, but Explicit is probably a better style.

Reply:
We could even add a eslint plugin to check for this.. https://github.com/buildo/eslint-plugin-fat-arrow-same-line